docker-compose.test.yml

```diff
@@ -41,6 +41,12 @@ services:
     build:
       context: ./app-backend/tile
       dockerfile: Dockerfile
+      
+  backsplash:
```

Comment:
If I follow, that might be a change to suggest here: https://github.com/raster-foundry/raster-foundry/pull/3954/files#diff-4e5e90c6228fd48698d074241c2ba760

Reply:
I think it is more a concern of this PR because it prevents publishing.

In this case, the responsibilities of both of these PRs are close enough that the line between them is blurred. Ideally, this one would have handled all of the changes specific to `docker-compose.test.yml`.